source/on.js

```diff
@@ -0,0 +1,30 @@
+import curryN from './internal/_curryN';
+
+
+/**
+ * Takes a binary curried function `f`, a unary function `g`, and two values.
+ * Applies `g` to each value, then applies the result of each to `f`.
+ *
+ * Also known as the P combinator.
+ *
+ * @func
+ * @memberOf R
+ * @category Function
+ * @sig ((a, a) -> b) -> (c -> a) --> c -> c -> b
+ * @param {Function} f a binary function
+ * @param {Function} g a unary function
+ * @param {any} a any value
+ * @param {any} b any value
+ * @return {any} The result of `f`
+ * @example
+ *
+ *      const eqBy = R.on((a, b) => a === b);
+ *      eqBy(R.prop('a'), {b:0, a:1}, {a:1}) //=> true;
+ *
+ *      const containsInsensitive = R.on(R.contains, R.toLower);
+ *      containsInsensitive('o', 'FOO'); //=> true
+ */
```

Comment:
I think it would be useful to add a `@symb` tag as well.  Here it would very much describe the implementation, but that's fine.  See [converge][co] or [map][ma] for examples.


  [co]: https://github.com/ramda/ramda/blob/c29a727b350ad1d37c428ec94ccf9f1ef31ff6c1/source/converge.js#L35
  [ma]: https://github.com/ramda/ramda/blob/c29a727b350ad1d37c428ec94ccf9f1ef31ff6c1/source/map.js#L41-L43

Reply:
If I understand correctly, this would be

    @symb R.on(f, g, a, b) = f(g(a), g(b))

?

Reply:
Yes, exactly.